src/duiker.py

```diff
@@ -36,7 +36,7 @@ def xdg_data_home(name: Optional[str] = None) -> pathlib.Path:
     Return the XDG Base Directory Specification data directory for a specific
     application, or the base directory itself.
     """
-    home = pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or '~/.local/share'))
+    home = pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or os.path.expanduser('~') + '/.local/share'))
```

Comment:
I will have to rollback and get the exception message, but it was a write permissions error on '~/.local/share' as if the path expansion was not occurring. Running on a checkout of master.